api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>For guidance, it would be helpful to have a bit more text. For example:</p><p> </p><p>"In this first example, we'll investigate weather conditions for regions within in Salta, a major soybean producing region of Argentina. In this step, we'll identify the IDs for all of the counties within Salta. If we wish to discover anomalies in another region, we can simply change the region_id to that of our region of interest."</p><p> </p><p>And I think we discussed making salta_region_id more generic. Maybe just region_id or something like region_is_sample_one so it doesn't clash with the Brazil script later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>